chainstate/processor.go

```diff
@@ -56,46 +56,72 @@ func (p *BloomProcessor) Logger() Logger {
 	return p.logger
 }
 
-// GetHash get the hash of the current filter
+// GetHash get the hash of the current Filter
```

Comment:
The word "Filter" vs "filter" in all the comments... looks like a refactor might have caused this

Reply:
Fixed